Dockerfile

```diff
@@ -16,20 +16,20 @@ ENV CONFIG_PATH="/opt/janus/etc/janus"
 # docker build arguments
 ARG BUILD_SRC="/usr/local/src"
 
-ARG JANUS_VERSION="v1.1.4"
-ARG JANUS_LIBNICE_VERSION="0.1.21"
+ARG JANUS_VERSION="v1.2.4"
+ARG JANUS_LIBNICE_VERSION="0.1.22"
 ARG JANUS_PAHO_MQTT_VERSION="v1.3.12"
 ARG JANUS_RABBITMQ_VERSION="v0.13.0"
-ARG JANUS_LIBSRTP_VERSION="2.5.0"
+ARG JANUS_LIBSRTP_VERSION="2.6.0"
 ARG JANUS_USRSCTP_VERSION="0.9.5.0"
-ARG JANUS_BORINGSSL_VERSION="e9f816b12b3e68de575d21e2a9b7d76e4e5c58ac"
-ARG JANUS_LIBWEBSOCKETS_VERSION="v4.3.2"
+ARG JANUS_BORINGSSL_VERSION="0568c2c1dbff4e1de4d5a63fbaf7d13925df27fa"
+ARG JANUS_LIBWEBSOCKETS_VERSION="v4.3.3"
 
 ARG JANUS_WITH_POSTPROCESSING="1"
 ARG JANUS_WITH_BORINGSSL="1"
 ARG JANUS_WITH_DOCS="0"
 ARG JANUS_WITH_REST="1"
-ARG JANUS_WITH_DATACHANNELS="0"
+ARG JANUS_WITH_DATACHANNELS="1"
```

Comment:
Why do we need this?
I thought we were already using the textroom plugin for the Co-browsing feature (to transmit click coordinates to highlight a point on the shared web page)?

Reply:
The textroom plugin uses data channels.

I guess the last built image had the data channels enabled but nothing was committed in the repo

Reply:
Exactly.
![image](https://github.com/user-attachments/assets/24847967-6608-4435-9f1a-6beab38d6a9a)